docs/specification/current.md

```diff
@@ -655,7 +641,7 @@ Field | Optionality | Type | Description
 `card` | REQUIRED | *string* | The `card.uuid` from the CDS Hooks response. Uniquely identifies the card.
 `outcome` | REQUIRED | *string* | A value of `accepted` or `overridden`.
 `acceptedSuggestions` | CONDITIONAL | *array* | An array of json objects identifying one or more of the user's **AcceptedSuggestion**s. Required for `accepted` outcomes.
-`overrideReason` | OPTIONAL | *object* | A json object identifying the **OverrideReason** selected by the user.
+`overrideReason` | OPTIONAL | *object* of **OverrideReason** | A json object capturing the override reason as well as any comments entered by the user.
```

Comment:
```suggestion
`overrideReason` | OPTIONAL | **Coding** | A json object capturing the **OverrideReason** as well as any comments entered by the user.
```

Reply:
This is not correct. The OverrideReason here is a type which contains a reason of type Coding, and a userComment.

Reply:
I updated this to just be **OverrideReason**, without the *object* of prefix.

Reply:
Ah, sorry, I got confused. Good catch.